tests/unit/lms/services/application_instance_getter_test.py

```diff
@@ -11,15 +11,20 @@
 
 
 class TestApplicationInstanceGetter:
-    def test_developer_key_returns_the_developer_key(self, ai_getter):
-        assert ai_getter.developer_key("TEST_CONSUMER_KEY") == "TEST_DEVELOPER_KEY"
+    def test_developer_key_returns_the_developer_key(self, ai_getter, pyramid_request):
+        assert (
+            ai_getter.developer_key(pyramid_request.lti_user.oauth_consumer_key)
+            == "TEST_DEVELOPER_KEY"
+        )
```

Comment:
I think this is because there's [another PR awaiting re-review](https://github.com/hypothesis/lms/pull/1637) that changes `ApplicationInstanceGetter` so that it _is_ specific to the Pyramid request LTI user consumer key (at which point the consumer key arg to this method goes away). So if I changed this test now, I'd immediately have to change it back again

Reply:
This really is as easy as just reverting the entire file though, so I've gone ahead and done it anyway

Reply:
Aw right, gotcha. The PR train will destroy you!